CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ This file summarizes **notable** changes for each release, but does not describe
 
 ----
 
+## v0.7.0 (2022-03-21)
+
+This is the spiritual successor to v0.5.0.
+
+### Breaking changes
+
+* We now use the upstream WebSocket model made public in http4s-0.23.11.
+
+### Dependency updates
+
+* scala-2.12.15, scala-2.13.8, scala-3.1.1
+* cats-2.7.0
+* cats-effect-3.2.5
```

Comment:
Latest CE is 3.3.8.
https://github.com/typelevel/cats-effect/releases/tag/v3.3.8

Reply:
Ah yes, that was fixed in #596